### ISSUE TITLE
Unexported intent filter error form Google Play console

### ISSUE THREAD

**Issue by marchare555** [renpy/renpy#3937]
I get the following error when uploading my game build on the Google Play console:

`You uploaded an APK or Android App Bundle which has an activity, activity alias, service or broadcast receiver with intent filter, but without 'android:exported' property set. This file can't be installed on Android 12 or higher. See: developer.android.com/about/versions/12/behavior-changes-12#exported`

If I upload The Question as a test, the build ships properly. Not sure why my own game gets that error and not The Question.

Not sure it that helps, but android.txt shows this:
```
> Task :renpyandroid:compileReleaseJavaWithJavac
Note: C:\Users\pikachu\Desktop\renpy-7.5.3-sdk\rapt\project\renpyandroid\src\main\java\org\renpy\android\PythonSDLActivity.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\Users\pikachu\Desktop\renpy-7.5.3-sdk\rapt\project\renpyandroid\src\main\java\org\renpy\android\PythonSDLActivity.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details

WARNING: Compatible side by side NDK version was not found for android.ndkVersion '21.3.6528147'
Compatible side by side NDK version was not found for android.ndkVersion '21.3.6528147'
```

Moreover, this happens across several Ren'py prereleases and nightly I tried over the past few days, as well as several different Java versions I installed.

Is it an issue with my game's code, or with RAPT?

**Comment by mattmurdock0010** [comment]
android:exported missing on app-AndroidManifest.xml line 33.

```
{% if (config.store in ("amazon", "all" )) %}
    <receiver android:name="com.amazon.device.iap.ResponseReceiver"
      android:permission="com.amazon.inapp.purchasing.Permission.NOTIFY" android:exported="true">
      <intent-filter>
        <action android:name="com.amazon.inapp.purchasing.NOTIFY" />
      </intent-filter>
    </receiver>
{% endif %}
```

**Comment by renpytom** [comment]
This was fixed by renpy/renpy-build@dcf65efe41a726bd6b0865953be301208d6b5a8b .
